roles/jd-server/src/lib/mempool/mini_rpc_client.rs

```diff
@@ -0,0 +1,207 @@
+// TODO
```

Comment:
put all this mini-rpc client in a specific crate under utils

Reply:
Add some tests

Reply:
will do

Reply:
I moved the client within `utils` in the last force push. What tests do you suggest?